sdk/keyvault/azure_security_keyvault_keys/Cargo.toml

```diff
@@ -0,0 +1,16 @@
+[package]
+name = "azure_security_keyvault_keys"
+version = "0.0.1"
```

Comment:
@jhendrixMSFT I thought we were starting things at 0.1.0? Seems odd to start with a PATCH release.

Reply:
The version is parameterized, so whatever the caller specifies is what it starts at. Where did the `0.0.1` come from?

Reply:
The 0.0.1 came from the specs repo. You can blame me